### ISSUE TITLE
Add support for Mac OS

### ISSUE THREAD

**Issue by Akasurde** [adtac/climate#11]

**Comment by adtac** [comment]
You'd really be better off with https://github.com/guarinogabriel/Mac-CLI for a mac. I know it's still unix, but I'm not sure how compatible climate is with a mac. But I'm keeping this issue open in case someone else wants to discuss this.

**Comment by Igosuki** [comment]
The only thing I can't seem to install for mac is sensors and upower, but the question is more about compatibility rather than being able to compile stuff.

**Comment by adtac** [comment]
Yeah. Some of these tools are very Linux specific (especially hardware related tools like battery), so I'm not really sure how to do this with a Mac.
